docs/source/tutorials/parsl-rp/parsl-rp.ipynb

```diff
@@ -0,0 +1,276 @@
+   "id": "53195bcc",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2022-11-22T12:17:52.430749Z",
+     "start_time": "2022-11-22T12:17:52.422540Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "%%capture capt\n",
+    "\n",
+    "import os\n",
+    "\n",
+    "mdb_host = os.environ.get('MDB_SERVER', 'mongodb')\n",
+    "mdb_port = os.environ.get('MDB_PORT',   '27017')\n",
+    "mdb_name = os.environ.get('MDB_NAME',   'guest')\n",
+    "mdb_pswd = os.environ.get('MDB_PSWD',   'guest')\n",
+    "mdb_dtbs = os.environ.get('MDB_DTBS',   'default')\n",
+    "\n",
+    "%env RADICAL_PILOT_DBURL=mongodb://$mdb_name:$mdb_pswd@$mdb_host:$mdb_port/$mdb_dtbs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bbf6a6d1",
+   "metadata": {},
+   "source": [
+    "# MPI Nwchem Example:\n",
+    "\n",
+    "The following example application shows the execution of MP2 geometry optimization followed by a CCSD(T) energy evaluation at the converged geometry. A Dunning correlation-consistent triple-zeta basis is used. The default of Cartesian basis functions must be overridden using the keyword spherical on the BASIS directive. The 1s core orbitals are frozen in both the MP2 and coupled-cluster calculations (note that these must separately specified)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "52f78a41",
+   "metadata": {},
+   "source": [
+    "First, we need to write the `nwchem` example to a file so we can use it provide it as an input to the `nwchem` executable later."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6980a500",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "input = \"\"\"\n",
+    "start n2\n",
+    "\n",
+    "geometry\n",
+    "  symmetry d2h\n",
+    "  n 0 0 0.542\n",
+    "end\n",
+    "\n",
+    "basis spherical\n",
+    "  n library cc-pvtz\n",
+    "end\n",
+    "\n",
+    "mp2\n",
+    "  freeze core\n",
+    "end\n",
+    "\n",
+    "task mp2 optimize\n",
+    "\n",
+    "ccsd\n",
+    "  freeze core\n",
+    "end\n",
+    "\n",
+    "task ccsd(t)\n",
+    "\"\"\"\n",
+    "nwchem_input = '{0}/{1}'.format(os.getcwd(), 'mp2_optimization.nw')\n",
+    "with open(nwchem_input,\"w+\") as f:\n",
+    "    f.writelines(input)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "39d40bda",
+   "metadata": {},
+   "source": [
+    "Now, we import parsl Python module in our application and the RadicalPilotExecutor"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "77692350",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2022-11-22T12:17:57.635805Z",
+     "start_time": "2022-11-22T12:17:57.340827Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import parsl\n",
+    "import radical.pilot as rp\n",
+    "\n",
+    "from parsl.config import Config\n",
+    "from parsl.app.app import python_app, bash_app\n",
+    "from parsl.executors import RadicalPilotExecutor"
```

Comment:
`os` and `python_app` can be dropped here (`os` is already loaded and `python_app` is not used)

Reply:
fixed